internal/config/supervisor/config.go

```diff
@@ -76,6 +76,9 @@ func FromPath(path string) (*Config, error) {
 	if err := validateEndpoint(*config.Endpoints.HTTP); err != nil {
 		return nil, fmt.Errorf("validate http endpoint: %w", err)
 	}
+	if err := validateAdditionalHTTPEndpointRequirements(*config.Endpoints.HTTP, config.AllowExternalHTTP); err != nil {
```

Comment:
If `config.AllowExternalHTTP` is a string, then we need to validate it.

Reply:
Updated in 51c527a965a743f89347420e6a5fe075a59ef3f0.